### PR TITLE
YLP-960 limit number of automatic emails sent for the same account

### PR DESCRIPTION
**Pull request by CedricPerotto**

### CODE REVIEW

**Review by yvon-dblg — COMMENTED** [review]
If we have more than 1 hydrophone process in k8s, it may not work as expected ?

**Comment by CedricPerotto** [comment]
> If we have more than 1 hydrophone process in k8s, it may not work as expected ?

No problems in my local stack... Maybe try with 1.7.4-rc1 published version.

File: api/hydrophoneApi.go [hunk 1]
```diff
@@ -487,3 +491,36 @@ func (a *Api) isAuthorizedUser(tokenData *token.TokenData, userId string) bool {
...
+			}
+		}
+	} else {
+		count, err = a.Store.CountLatestConfirmations(ctx, confirm, createdSince)
+		a.logger.Printf("verifySendAttempts::count:%v for %v", count, confirm)
+	}
+	if err != nil {
+		return false, 0, err
+	}
+	if count > a.Config.ConfirmationAttempts {
```
Comment (ydrisR) [inline api/hydrophoneApi.go]:
```suggestion
	if count >= a.Config.ConfirmationAttempts {
```

Comment (ydrisR) [reply api/hydrophoneApi.go]: I would have limited the number of attempts to what we specifiy in the configuration.
Not critical but a bit missleading I think

**Review by ydrisR — APPROVED** [review]
LGTM with one exception... But I let you decide if it's worth to change or not :)

**Pull request merged**